extensions/smallrye-health/runtime/src/main/java/io/quarkus/smallrye/health/runtime/SmallRyeHealthHandlerBase.java

```diff
@@ -18,6 +20,16 @@
 
     protected abstract SmallRyeHealth getHealth(SmallRyeHealthReporter reporter, RoutingContext routingContext);
 
+    private static final Map<String, String> corsHeaders;
+
+    static {
+        corsHeaders = new HashMap<>();
+        corsHeaders.put("Access-Control-Allow-Origin", "*");
```

Comment:
Do we really want to allow everything and not make it configurable?

That seems a bit dangerous even if health information is probably not that critical. Maybe it's not worth it, I'll let you judge of that :).

Reply:
Yeah - we already have a CORS filter which should be usable ?